js/rooms.js

```diff
@@ -78,4 +104,18 @@ $(document).ready(function() {
 		}
 	};
 
+	$(document).click(function(e) {
```

Comment:
Yes we should – the menu is also implemented in Calendar (@georgehrke @raghunayyar) and News (@bernhardposselt) ;)